config/default/system.image.yml

```diff
@@ -1,3 +1,3 @@
 _core:
   default_config_hash: durWHaKeBaq4d9Wpi4RqwADj1OufDepcnJuhVLmKN24
-toolkit: gd
+toolkit: imagemagick
```

Comment:
Out of curiosity, is imagemagick in use on .gov?

Reply:
It's installed in Dockerfile so it's available